diff --git a/dist/index.js b/dist/index.js
index 14b86a68c377c546e4030009546fa40768378564..81f8723adf4fb7de682877e7428148c715886c50 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1,7 +1,7 @@
 Object.defineProperties(exports, { __esModule: { value: true }, [Symbol.toStringTag]: { value: 'Module' } });
 const require_runtime = require('./_virtual/_rolldown/runtime.js');
 const require_utils = require('./utils.js');
-const require_index = require('./node_modules/fast-decode-uri-component/index.js');
+const require_index = require('fast-decode-uri-component');
 let elysia = require("elysia");
 
 //#region src/index.ts
diff --git a/dist/index.mjs b/dist/index.mjs
index 982fa37c5d0a381a8025af2154e81cf78e838844..d13d36a5bb7100b484a95c13d5d8baeebef41b2b 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -1,10 +1,9 @@
-import { __toESM } from "./_virtual/_rolldown/runtime.mjs";
 import { LRUCache, alreadyCachedDownstream, fileExists, generateETag, getBuiltinModule, getFile, getFileStats, isBun, listFiles } from "./utils.mjs";
-import { require_fast_decode_uri_component } from "./node_modules/fast-decode-uri-component/index.mjs";
+import fastDecodeURIComponent from "fast-decode-uri-component";
 import { Elysia, NotFound } from "elysia";
 
 //#region src/index.ts
-var import_fast_decode_uri_component = /* @__PURE__ */ __toESM(require_fast_decode_uri_component());
+var import_fast_decode_uri_component = { default: fastDecodeURIComponent };
 async function staticPlugin({ assets = "public", prefix = "/public", staticLimit = 1024, alwaysStatic = process.env.NODE_ENV === "production", ignorePatterns = [
 	".DS_Store",
 	".git",
